language/vm/vm-runtime/vm-runtime-types/src/value.rs

```diff
@@ -362,6 +363,16 @@ impl Value {
     pub fn pretty_string(&self) -> String {
         self.0.pretty_string()
     }
+
+    pub fn is_valid_script_arg(&self, sig: &SignatureToken) -> bool {
```

Comment:
this is really a hack but again part of the more logical clean up of the Libra and the Move layer